engine/api/environment_import.go

```diff
@@ -73,7 +73,6 @@ func (api *API) postEnvironmentImportHandler() service.Handler {
 			if sdk.ErrorIsUnknown(globalError) {
 				return globalError
 			}
-			log.Warning("%v", globalError)
```

Comment:
if you want to remove it, you can remove it from:
https://github.com/ovh/cds/blob/7ea89dd5a82080f5e80716df1a3253cb81d1ea30/engine/api/pipeline_import.go#L99

Reply:
fixed